Makefile

```diff
@@ -64,7 +64,7 @@ EXTRA_GOFLAGS ?=
 MAKE_VERSION := $(shell $(MAKE) -v | head -n 1)
 MAKE_EVIDENCE_DIR := .make_evidence
 
-ifeq ($(RACE_ENABLED),true)
+ifneq ($(RACE_ENABLED),)
```

Comment:
`export RACE_ENABLED=true` worked on my PC, but the CI somehow set it different ... so i moved back to just test if RACE_ENABLED is set ...